paasta_tools/cli/cmds/remote_run.py

```diff
@@ -52,50 +77,36 @@ def add_remote_run_args(parser):
         default=DEFAULT_SOA_DIR,
     )
     parser.add_argument(
-        '--json-dict',
-        help='When running dry run, output the arguments as a json dict',
+        '-v', '--verbose',
+        help='Show Docker commands output',
         action='store_true',
-        dest='dry_run_json_dict',
-    )
-    parser.add_argument(
-        '-C', '--cmd',
-        help=('Run Docker container with particular command, '
-              'for example: "bash". By default will use the command or args specified by the '
-              'soa-configs or what was specified in the Dockerfile'),
         required=False,
-        default=None,
+        default=True,
     )
     parser.add_argument(
-        '-i', '--instance',
-        help=("Simulate a docker run for a particular instance of the service, like 'main' or 'canary'"),
+        '-R', '--run-id',
+        help='Identifier to assign/refer to individual task runs',
+        action='store',
         required=False,
         default=None,
-    ).completer = lazy_choices_completer(list_instances)
-    parser.add_argument(
-        '-v', '--verbose',
-        help='Show Docker commands output',
-        action='store_true',
-        required=False,
-        default=True,
     )
     parser.add_argument(
         '-d', '--dry-run',
         help='Don\'t launch the task',
-        action='store_true',
+        action='count',
```

Comment:
we had -D to mean dry-run for paasta_tools/cli/cmds/remote_run.py and -d - dry-run for paasta_tools/paasta_remote_run.py

now -D means --detach and -d is repeatable option, when passed once it goes to paasta_remote_run on master, more than once - to cli/cmds/remote_run